lambda-multi-svc/service-crud/pipeline_infrastructure/cloudformation.yaml

```diff
@@ -36,7 +36,7 @@ Resources:
                             "python3.8": {"python": "3.8"},
                             "java11": {"java": "openjdk11.x"},
                             "dotnetcore3.1": {"dotnet": "3.1"}
-                          }[service_instances[0].spec.lambda_runtime] | tojson | safe }},
+                          }[service_instances[0].outputs.LambdaRuntime] | tojson | safe }},
```

Comment:
why did `lambda_runtime` change to `LambdaRuntime`?

Reply:
`lambda_runtime` is an input, and currently we only make outputs available to the service instance.  So I put LambdaRuntime as an output in the instance cloudformation, which then becomes available to the pipeline.